org.opent2t.sample.thermostat.superpopular/test/thermostatTests.js

```diff
@@ -0,0 +1,118 @@
+'use strict';
+
+var OpenT2T = require('opent2t').OpenT2T;
+
+function runThermostatTests(translatorPath, mockDevice, test, testData) {
```

Comment:
I'll do the investigation as a new task after this goes in.

---
In reply to: [82072262](https://github.com/openT2T/translators/pull/106#discussion_r82072262) [](ancestors = 82072262)